docs/docs/migrate/keptnapp/assets/keptnapp-migrated.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: "some-keptn-app"
   namespace: "my-app-ns"
   labels:
```

Comment:
pls also remove this field since it doesn't have any content anymore. otherwise this would be a YAML syntax error 😅

Reply:
done.  We need to look at this again -- if the old KeptnApp was autogenerated then edited for pre/post-deployment tasks//evaluations, it would have the managed-by annotation as a label.  This is explained on the ref page but not sure how to handle it for migration guide -- let me think about it as I do read-through/xrefs path.